recipes/mysterymaster/meta.yaml

```
@@ -0,0 +1,34 @@
+{% set version = "0.0.4" %}
+{% set base = "https://bitbucket.org/NPC239/mysterymaster/downloads/" %}
+{% set sha256 = "8895356da7ad46724b68bc931eff09b16b61eb76fd5229f8f467570b12354014" %}
+
+package:
+  name: mysterymaster
+  version: '{{version}}'
+
+source:
+  url: {{base}}/mysterymaster-{{version}}.tar.gz
+  sha256: '{{sha256}}'
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('mysterymaster', max_pin="x.x") }}
+
+requirements:
+  host:
+    - openjdk >=11
+  run:
+    - openjdk >=11
+
+test:
+  commands:
+    - FindBarcodes -h >/dev/null
```

```suggestion
    - FindBarcodes -h 2>&1 | grep XXX
```
Please replace XXX with something meaningful